src/main.rs

```diff
@@ -45,26 +67,40 @@ pub(crate) struct Indexer {
     pub client: ArbitrumClient,
     /// The decryption key
     pub decryption_key: DecryptionKey,
+    /// A connection to the DB
+    pub db_conn: PgConnection,
 }
 
 impl Indexer {
     /// Constructor
-    pub fn new(client: ArbitrumClient, decryption_key: DecryptionKey) -> Self {
+    pub fn new(
+        client: ArbitrumClient,
+        decryption_key: DecryptionKey,
+        db_conn: PgConnection,
+    ) -> Self {
         Indexer {
             client,
             decryption_key,
+            db_conn,
         }
     }
 }
 
 /// Main
 #[tokio::main]
 async fn main() -> Result<(), Box<dyn Error>> {
+    setup_system_logger(LevelFilter::INFO);
```

Comment:
can be a TODO for now, but would like to apply the datadog log formatter as we do in the relayer in anticipation of us piping logs to datadog.